src/pybind/mgr/cephadm/upgrade.py

```diff
@@ -530,17 +543,18 @@ def _do_upgrade(self):
                 else:
                     logger.info('Upgrade: Updating %s.%s' %
                                 (d.daemon_type, d.daemon_id))
+                action = 'Upgrading' if d_entry[1] else 'Redeploying'
```

Comment:
I think this is backward... d_entry[1] is true for the correct digest but wrong deployer

Reply:
oops